BridgeCareApp/BridgeCare/Controllers/SectionGeoLocatorController.cs

```diff
@@ -6,12 +6,12 @@
 
 namespace BridgeCare.Controllers
 {
-    public class LatitudeLongitudeSectionController : ApiController
+    public class SectionGeoLocatorController : ApiController
     {
         private readonly BridgeCareContext db;
-        private readonly ILatitudeLongitudeSection LatitudeLongitude;
+        private readonly ISectionGeoLocator LatitudeLongitude;
```

Comment:
Change the name of the variable to reflect what it is doing. LatitudeLongitude is not reflective of what the variable is doing.